### PR TITLE
Allow to load multiple animations at once

### LINKED ISSUE #30227: Abillity to open multiple animations at once
**Godot version:** v3.1.1.stable.official

**OS/device including version:** Linux Mint 19.1

**Issue description:**

It would be nice if you could select multiple animation files when opening animations on an AnimationPlayer using Animation -> Load.

### PR DESCRIPTION
**Pull request by KoBeWi**
Closes #30227

### CODE REVIEW

File: editor/plugins/animation_player_editor_plugin.cpp [hunk 1]
```diff
@@ -733,51 +731,51 @@ void AnimationPlayerEditor::_animation_edit() {
...
+		}
+		if (file.find_last("\\") != -1) {
 
-				_animation_save_in_path(current_res, p_file);
-			}
+			file = file.substr(file.find_last("\\") + 1, file.length());
 		}
+
+		if (file.find(".") != -1)
+			file = file.substr(0, file.find("."));
```
Comment (Calinou) [inline editor/plugins/animation_player_editor_plugin.cpp:768]:
```suggestion
		if (file.find(".") != -1) {
			file = file.substr(0, file.find("."));
		}
```

File: editor/plugins/animation_player_editor_plugin.cpp [hunk 2]
```diff
@@ -733,51 +731,51 @@ void AnimationPlayerEditor::_animation_edit() {
 	}
 }
 
-void AnimationPlayerEditor::_dialog_action(String p_file) {
+void AnimationPlayerEditor::_save_animation(String p_file) {
```
Comment (Calinou) [inline editor/plugins/animation_player_editor_plugin.cpp:735]:
```suggestion
```

**Review by Calinou — APPROVED** [review]
Should be good to merge after rebasing and applying code style changes.

**Comment by mhilbrunner** [comment]
@KoBeWi Sorry, but this needs to be rebased - otherwise looks good to go. Feel free to ping me for a merge if you rebase it :)

**Comment by KoBeWi** [comment]
@mhilbrunner

**Comment by mhilbrunner** [comment]
Thanks! And thanks for the rebase :) Tested, works.

**Comment by fire** [comment]
TL;DR

In the file browser select multiple .anim and they are all loadable in the animation load dialog in the Animation Player.
